ch_collections/heritage_services/roles/nfs/tasks/main.yml

```diff
@@ -23,6 +24,7 @@
     owner: root
     group: root
     mode: '0755'
+  when: install_watcher_service
```

Comment:
Don't install the service when `install_watcher_service` is false, in this case it won't be configured or enabled so we shouldn't leave it existing in a non funcitonal state.